jest/private/jest_test.bzl

```diff
@@ -45,9 +45,17 @@ _attrs = dicts.add(js_binary_lib.attrs, {
         allow_single_file = True,
         default = Label("//jest/private:jest_config_template.mjs"),
     ),
+    # Earlier versions of Bazel expect this attribute to be present.
+    # We use a no-op because jest itself generates the coverage.
```

Comment:
@thesayyn Do you know of any issues or comments I can link to for this statement?

Reply:
Nevermind, found https://github.com/bazelbuild/bazel/issues/13978.